lib/screens/splash/splash_screen.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:solaris_structure_1/router/routing_constants.dart';
+import 'package:solaris_structure_1/animations/splashscreen_fadeout_animation.dart';
+
+class SplashScreen extends StatefulWidget {
+  const SplashScreen({super.key});
+
+  @override
+  State<SplashScreen> createState() => _SplashScreenState();
+}
+
+class _SplashScreenState extends State<SplashScreen> {
+  @override
+  void initState() {
+    super.initState();
+    Timer(const Duration(seconds: 3), () {
```

Comment:
How did you pick `3` as time to wait until redirect? (wait time is generally not appreciated by users).

Also, `3` seems like a magic number, what do you say to move it to a constant? Maybe with the constant name it would be more clear why we needed and why this duration.
@szaszgyozo